src/internal/innies-and-outies.ts

```diff
@@ -0,0 +1,114 @@
+/** @prettier */
+import { Subscription } from './Subscription';
+import { Subscriber } from './Subscriber';
+import { Observable } from './Observable';
+import { subscribeTo } from './util/subscribeTo';
+
+export interface ISimpleOuterSubscriber<T> {
```

Comment:
I trivial nit - as the interface is used only in this file - but I'd go with the TypeScript convention of using a `Like` suffix over an `I` prefix for interfaces/shapes. IIRC, we do this for the library's exported types. Whether or not it's done here doesn't really matter, though.

Reply:
Yeah, I don't like the `I`, much either, I guess.